app/javascript/components/timeline-options/timeline-helper.js

```diff
@@ -43,7 +43,7 @@ const smartAndOrStatements = (group, array) => {
 export const buildUrl = (values) => {
   // TODO: Different timeline show different data, ensure all necesary data is pulled
   let url = `/api/event_streams?limit=5000&offset=0&expand=resources`;
-  url += `&attributes=group,group_level,group_name,id,event_type,ems_id,type,timestamp,created_on,host.name,source`;
+  url += `&attributes=group,group_level,group_name,id,event_type,ems_id,type,timestamp,created_on,host,source`;
```

Comment:
I noticed you removed host.name - won't that make it a lot more expensive if it's returning the entire host object?

Reply:
Yes, that was causing the issue 4) above.  That was removed and I added issue 7) to add the proper ems_id/host_id filter[] as we're not filtering properly right now.

Reply:
I was thinking of either a virtual column or something that would make it more reliable as the `host.name` sometimes comes back and others it doesn't.  It makes the logic needlessly complex in the UI.

Example API response without host.name:

```
        {
            "href": "http://localhost:3000/api/event_streams/48428",
            "id": "48428",
            "event_type": "MODIFY_URI",
            "ems_id": "2",
            "type": "EmsEvent",
            "timestamp": "2022-10-05T04:00:31Z",
            "created_on": "2022-10-05T03:29:55Z",
            "source": "IBM_POWER_HMC",
            "host_id": "1",
            "group": "configuration",
            "group_level": "detail",
            "group_name": "Configuration/Reconfiguration"
        },
```

and with a host.name:

```

        {
            "href": "http://localhost:3000/api/event_streams/67912",
            "id": "67912",
            "event_type": "MODIFY_URI",
            "ems_id": "2",
            "type": "EmsEvent",
            "timestamp": "2022-10-06T09:58:54Z",
            "created_on": "2022-10-06T09:28:19Z",
            "source": "IBM_POWER_HMC",
            "host_id": "7",
            "group": "configuration",
            "group_level": "detail",
            "group_name": "Configuration/Reconfiguration",
            "host": {
                "name": "porthos"
            }
        },
```

Reply:
I've opened this PR to add virtual columns to address this problem:  https://github.com/ManageIQ/manageiq/pull/22398